### ISSUE TITLE
[RyuJIT/ARM32] Assertion failure in lsra.cpp

### ISSUE THREAD

**Issue by hseok-oh** [dotnet/coreclr#12602]
Assertion message

```
Assert failure(PID 842 [0x0000034a], Thread: 842 [0x034a]): Assertion failed '((consume == 0) && (produce == 0)) || (ComputeAvailableSrcCount(tree) == consume)' in 'JitTest.LCS:Main():int' (IL size 279)
               
    File: /opt/code/github/hqueue/coreclr/src/jit/lsra.cpp Line: 3620
    Image: /home/dotnet_jenkins/workspace/hqueue/Run_coreclr_tests_Checked_ryujit_nyi_all/runtime/tests/Tests/coreoverlay/corerun
```

Tests
JIT/Methodical/refany/_il_dbglcs/_il_dbglcs.sh
JIT/Methodical/refany/lcs/lcs.sh

**Comment by YongseopKim** [comment]
I faced this issue. I modified [here](https://github.com/YongseopKim/coreclr/commit/fdc3b032e522512a71cc29eb6385b36d9720e35f#diff-498fd4859cb147c9bf082f5ebf32ca8fR1402)

**Comment by hseok-oh** [comment]
Fixed by #12628
